doc/themes/the-things-stack/layouts/shortcodes/proto/message.html

```diff
@@ -40,6 +40,16 @@ <h3 id="message:{{ .name }}">Message <code>{{ .name }}</code></h3>
             <p><code>{{ $rule }}</code>{{ if not (eq $value true)}}: <code>{{ $value }}</code>{{ end }}</p>
           {{- end }}
         {{ end }}
+        {{- if .map_key }}
+          {{- range $rule, $value := .map_key.rules }}
+            <p><code>{{ $rule }}</code> (key){{ if not (eq $value true)}}: <code>{{ $value }}</code>{{ end }}</p>
```

Comment:
```suggestion
            <p>Key: <code>{{ $rule }}</code>{{ if not (eq $value true)}}, value: <code>{{ $value }}</code>{{ end }}</p>
```

Reply:
That would render something like

> Key: `max_length`, value: `42`. Don't think that's what we want.

Reply:
The (key) in brackets is weird too :shrug: . I guess we can rather remove it altogether.